ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -107,6 +107,6 @@ def reset(self):
             self._runtime_metrics = RuntimeMetrics()
 
     def __repr__(self):
-        return 'RuntimeMetricsCollectorWorker({})'.format(
+        return 'RuntimeWorker(runtime_metrics={})'.format(
```

Comment:
Use `self.__class__.__name__` so it is never out-of-date and works with subclasses.